lib/rouge/lexers/php.rb

```diff
@@ -137,12 +143,17 @@ def self.detect?(text)
         end
 
         rule %r/(true|false|null)\b/, Keyword::Constant
+        rule /(?:void|\??(?:int|float|bool|string|iterable))\b/i, Keyword::Type
```

Comment:
Two things:

1. Can you preface the regex with `%r`? This keeps it consistent with the way the other rules are written (we do this to conform with Rubocop's rules).

2. So a legitimate type is `?int` or `?string`?

Reply:
1. an oversight
2. [yes](https://www.php.net/manual/en/migration71.new-features.php) (the `?` at the beginning allows the value to be `NULL`)